examples/TID_model_custom_property/Modelling_TID_TNID_example_notebook.ipynb

```diff
@@ -0,0 +1,471 @@
+{
```

Comment:
<p>This cell fails for me with "TID not defined"</p><p> </p><p>Also the plot looks like a gaussian but shouldn't this be the failure probability (above you wrote "g(x) is the Probability Distribution Function of the devices' failure doses.</p><p>" ? If so, that should monotonically increasing with x, no?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
You're right, it is a density function. It's a typo